packages/ember-metal/lib/mixin.js

```diff
@@ -702,7 +702,6 @@ Alias.prototype = new Descriptor();
   @method aliasMethod
   @for Ember
   @param {String} methodName name of the method to alias
-  @private
```

Comment:
Every documentation block must include @public or @private (to ensure it is super clear what is and is not private).  Our test suite enforces this.

This @private was likely added while cleaning up all blocks. 

Can you add @public here instead?